src/pcapi/infrastructure/repository/venue/venue_with_offerer_name/venue_with_offerer_name_sql_repository.py

```diff
@@ -36,6 +37,9 @@ def get_by_pro_identifier(
             else:
                 query = query.filter(UserOfferer.validationToken.isnot(None))
 
+        if active_offerers_only:
+            query = query.filter(Offerer.isActive)
```

Comment:
```suggestion
            query = query.filter(Offerer.isActive.is_(True))
```
Je ne sais pas si c'est justifié ici, mais j'avais lu dans la doc SQL Alchemy que l'utilisation de `.is_()` dans le cas de valeurs booléennes pouvait être préférable https://docs.sqlalchemy.org/en/13/core/sqlelement.html?highlight=is_#sqlalchemy.sql.expression.ColumnElement.is_